SDWebImage/Core/SDImageCacheConfig.m

```diff
@@ -34,7 +34,7 @@ - (instancetype)init {
         _diskCacheWritingOptions = NSDataWritingAtomic;
         _maxDiskAge = kDefaultCacheMaxDiskAge;
         _maxDiskSize = 0;
-        _diskCacheExpireType = SDImageCacheConfigExpireTypeModificationDate;
```

Comment:
It's ok to change default value, but this PR will marked as `Behavior Changes` and I'll point this on release changelog

Actually I don't know why the original author use `write` and expire type, since it's not actually LRU (we usually don't write to same cache key image)